bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -834,6 +854,8 @@ protected void release(boolean wait, boolean writer)
         } else {
           future = null;
         }
+      } else if (storage == null) {
+        resumeSuspendedClients(-1);
```

Comment:
Oh there was another change in the PR earlier in resume that provided an alternate part for spooling disabled case. It is not needed anymore.